### PR TITLE
Add a script to profile the conversion of a model

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: benchmarks/profile_conversion_time.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# coding: utf-8
```
Comment (guschmue) [inline benchmarks/profile_conversion_time.py:1]: We want to limit top level directories - can we stick this under tools?

File: ci_build/azure_pipelines/templates/unit_test.yml [hunk 2]
```diff
@@ -14,3 +14,14 @@ steps:
     condition: succeededOrFailed()
     env:
         CI_ONNX_OPSET: '${{ onnx_opset }}'
+
+  - bash: |
+      export TF2ONNX_TEST_BACKEND=$CI_ONNX_BACKEND
```
Comment (guschmue) [inline ci_build/azure_pipelines/templates/unit_test.yml:19]: Our ci pipleline is already slow, don't think we want to add benchmarks to it.

File: benchmarks/profile_conversion_time.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+"""
+Profiles the conversion of a Keras model.
+"""
+import fire
+import tensorflow as tf
+from tf2onnx import tfonnx
+from tensorflow.keras.applications import MobileNet
```
Comment (guschmue) [inline benchmarks/profile_conversion_time.py:8]: Iffy to depend on tf.keras since we must support older tf-1.x versions.

Comment (xadupre) [reply benchmarks/profile_conversion_time.py:8]: We can remove the script from CI, no issue with that, it takes 30 seconds. My plan was to only run it with the latest version of tensorflow. However, the script is failing and it was working a month ago. So I was wondering what change could have caused that. Either my script is wrong, either there is something not covered by the unit tests...

File: benchmarks/profile_conversion_time.py (hunk 1 above)
Comment (xadupre) [reply benchmarks/profile_conversion_time.py]: Done.

File: benchmarks/profile_conversion_time.py (hunk 3 above)
Comment (guschmue) [reply benchmarks/profile_conversion_time.py]: can we not start using fire ? We don't use it anywhere else and want to limit the dependencies.

**Comment by guschmue** [comment]
Isn't this PR seems to be included in here https://github.com/onnx/tensorflow-onnx/pull/1060 ?

**Comment by sdpython** [comment]
I merged it to check it was working. I was about to remove the merge.

File: ci_build/azure_pipelines/templates/unit_test.yml (hunk 2 above)
Comment (xadupre) [reply ci_build/azure_pipelines/templates/unit_test.yml]: Fixed

File: benchmarks/profile_conversion_time.py (hunk 3 above)
Comment (xadupre) [reply benchmarks/profile_conversion_time.py]: Dependency removed.

**Review by guschmue — APPROVED** [review]

**Pull request merged**